src/platforms/go/common/goroutines.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Goroutines
 sidebar_order: 5
+description: "Learn more about managing goroutines."
```

Comment:
I think we should rename (with a redirect) this section to "Concurrency", as it is about using the SDK when concurrency is at play. 

```suggestion
description: "Learn more about how to use the SDK from concurrent goroutines."
```

Reply:
Will do.